lib/analysis_options.yaml

```diff
@@ -1,6 +1,4 @@
 analyzer:
-  plugins:
-    - custom_lint
```

Comment:
I do not think we should change `analysis_options.yaml` here. The issue is about the `analysis_options_test.yaml` and its setup to work only for the `test` package of the end users.